src/olympia/activity/views.py

```diff
@@ -50,17 +53,23 @@ def check_object_permissions(self, request, obj):
 
     def get_serializer_context(self):
         ctx = super(VersionReviewNotesViewSet, self).get_serializer_context()
-        ctx['to_highlight'] = self.pending_queryset(
-            amo.LOG.DEVELOPER_REPLY_VERSION)
+        ctx['to_highlight'] = filter_queryset_to_pending_replies(
+            self.get_queryset())
         return ctx
 
-    def pending_queryset(self, log_type):
-        version_qs = self.get_queryset()
-        latest_reply = version_qs.filter(action=log_type.id).first()
-        if not latest_reply:
-            return version_qs
-        return version_qs.filter(created__gt=latest_reply.created)
-
+    def create(self, request, *args, **kwargs):
+        version = self.get_version_object()
+        if not version == version.addon.latest_or_rejected_version:
+            raise ParseError(
+                'Only latest versions of addons can have notes added.')
```

Comment:
I can't find any consistency with localising exception text content.    Most of the `raise ...` in Olympia aren't localised but there are some.